code/script.js

```diff
@@ -1,51 +1,316 @@
     `
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
+  chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  // here we call the function showMessage
+  showMessage("Welcome to dreamhoster.net, your service provider for professional hosting. What services are you interested in?", 'bot');
+}
+
+// 1a. all general service offers
+const serviceOffers = () => {
+  showMessage(`&bull; Domain registration<br>&bull; Web site hosting<br>&bull; Domain registration and hosting`, "bot");
+
+  // create 3 buttons
+  document.getElementById("input-wrapper").innerHTML = `
+  <div id="services-wrapper" style="display: flex; justify-content: space-evenly">
+    <button id="domain" value="domain">Domain<br>registration:<br>15 credits / year</button>
+    <button id="hosting" value="hosting">Hosting:<br>5  credits / month</button>
+    <button id="domainAndHosting" value="domainAndHosting">Registration<br>and Hosting:<br>15% discount</button>
+  </div>`;
+  // additional div for targeting content and serving as flex container, input-wrapper needs to ramain. css-styling only possible here inline, not in css file.
+  // do not nest ""
+
+  const servicesWrapper = document.getElementById("services-wrapper");
+
+  servicesWrapper.addEventListener("click", function (event) {
+    // do not target predefined input-wrapper for every query! target needs to be different, therefore add additional div in costum html
+    // console.log(event.target.value); -> visible after click
+    if (event.target.value == "domain") {
+      serviceChoice = `Domain registration only`;
+      FeeDomain = 15;
+      setTimeout(domainOffers, 1000);
+      // without timeout confirmation too late
+      // calling next question - 1b
+    } else if (event.target.value == "hosting") {
+      serviceChoice = "Hosting only";
+      // console.log("end");
+      FeeHosting = 5;
+      FeeHosting = FeeHosting * 12;
+      setTimeout(planSelection, 1000);
+    } else {
+      serviceChoice = "Domain and Hosting";
+      FeeHosting = 5;
+      FeeHosting = FeeHosting * 12;
+      FeeDomain = 15;
+      discount = 0.15;
+      setTimeout(domainOffers, 1000);
+      // without timeout confirmation too late
+      // calling next question - 1b
+    }
+    // DEAR EVALUATOR, I HAD NO CLUE HOW TO PULL OUT THE COND.STATEMENT AND OUTSOURCE IT SOMEWHERE ELSE... :(
+    // console.log(serviceChoice);
+    selectionConfirmation();
+
+    // console.log("end of services");
+  });
+}
+
+const selectionConfirmation = () => showMessage(`I pick: <br>${serviceChoice}`, "user");
```

Comment:
I'm wondering if you need the statements 'I pick' and 'I guess' in your user output for the chatbot? but this isnt code related :-)